tasks/test.js

```diff
@@ -78,6 +78,15 @@ module.exports = function (grunt) {
     'run:functionalTests',
   ]);
 
+  grunt.registerTask('test:ui:wdio', [
```

Comment:
Relying on grunt for executing `wdio` isn't a good idea. We're trying to move away from it where possible and this integration doesn't really add any value. Instead, I suggest removing this task and the scripts in `package.json`, and instead require executing `node scripts/wdio` directly. See https://github.com/elastic/kibana/issues/11095 for additional reasoning for relying on `node scripts/*` directly.